package.json

```diff
@@ -43,10 +43,10 @@
 		"update": "yarn up \"*\" -i"
 	},
 	"dependencies": {
+		"discord-api-types": "^0.31.2",
 		"dotenv": "^16.0.0",
 		"undici": "^5.0.0",
-		"ws": "^8.5.0",
-		"discord-api-types": "^0.31.2"
+		"ws": "^8.5.0"
```

Comment:
Could you revert these changes? Nothing was changed so it seems unnecessary to shuffle them around

Reply:
I'm pretty sure some lint got run and put in some ordering

Reply:
Yeah I wasn't sure why these committed, good catch.